### DISCUSSION TITLE
Github action GLOB event filtering DOES NOT work as expected

### DISCUSSION THREAD

**Discussion by tamipangadil** [community/community#26335] [Actions]
<p>I have a workflow to bump our repo tag and it will be triggered once changes are coming from any files under <code>production</code> folder. See my snippets below.</p>
<pre><code class="lang-auto"># bump-my-tag-version.yaml
name: Bump my tag version!

on:
  push:
    branches:
      - 'master'
    paths:
      - '**/production'
      - '!excluded_dir/production'

jobs:
  bump-version:
    runs-on: ubuntu-20.04
...

</code></pre>
<p>I have a few directories that has <code>production</code> folder as seen below.</p>
<pre><code class="lang-auto">excluded_dir/production
included_A/some_dir/production
included_B/some1/some2/production
included_C/some1/some2/some3/production
</code></pre>
<p>When I merge my changes to my main branch (<code>master</code>), it did not trigger the workflow. Which I expect it should because this filtering has already been implemented. See this <a href="https://github.blog/changelog/2019-09-30-github-actions-event-filtering-updates/" rel="noopener nofollow ugc">post</a> from GitHub.</p>
<p>Is this a bug or something is wrong with my implementation?</p>

**Comment by kingthorin** [comment]
<p>I believe you need asterisk or double asterisk on your globs right side,  because it isn’t about the path itself but rather things below that path. Ex: the files you’re changing.</p>

**Comment by tamipangadil** [comment]
<p>I’ll try this <a class="mention" href="/kingthorin">@kingthorin</a>. Thanks for the suggestion. I may have missed that part. I’ll let you know soon.</p>

**Comment by tamipangadil** [comment, marked as answer]
<p>Hey <a class="mention" href="/kingthorin">@kingthorin</a>, your suggestion works! Thank you so much!</p>
<p>Just to summarised the solution:</p>
<pre><code class="lang-auto"># bump-my-tag-version.yaml
name: Bump my tag version!

on:
  push:
    branches:
      - 'master'
    paths:
      - '**/production/**' # &lt;-- added 2 asterisks here
       - '!excluded_dir/production/**' # &lt;-- added 2 asterisks here

jobs:
  bump-version:
    runs-on: ubuntu-20.04
</code></pre>

**Comment by kingthorin** [comment]
<p>Great glad we got that sorted out.</p>
